docs/governance.rst

```diff
@@ -0,0 +1,253 @@
+  `https://github.com/ME-ICA/tedana <https://github.com/ME-ICA/tedana>`_
+
+Contributor
+  Person listed in the `all-contributors file`_.
+  The community decides on the content of this file using the same process as any
+  other change to the Repository (see below) allowing the meaning of "Contributor"
+  to evolve independently of the Decision-making rules.
+
+Maintainer
+  A Contributor responsible for the long term health of the project and the
+  community. Maintainers have additional rights (see Rules) helping them to
+  resolve conflicts and increase the pace of the development when necessary.
+  Any maintainer can self-remove themselves. Any contributor can become a
+  maintainer by request and with the support of the majority of the current
+  maintainers. Current Maintainers:
+
+  +-----------------------------------+-----------------+
+  | Name                              | Time commitment |
+  +===================================+=================+
+  | `Logan Dowdle`_ (@dowdlelt)       | 0.5h/week       |
+  +-----------------------------------+-----------------+
+  | `Elizabeth DuPre`_ (@emdupre)     | 0.5h/week       |
+  +-----------------------------------+-----------------+
+  | `Dan Handwerker`_ (@handwerkerd)  | 0.5h/week       |
+  +-----------------------------------+-----------------+
+  | `Ross Markello`_ (@rmarkello)     | 0.5h/week       |
+  +-----------------------------------+-----------------+
+  | `Taylor Salo`_ (@tsalo)           | 3h/week         |
+  +-----------------------------------+-----------------+
+  | `Joshua Teves`_ (@jbteves)        | 0.5h/week       |
+  +-----------------------------------+-----------------+
+  | `Eneko Uruñuela`_ (@eurunuela)    | 0.5h/week       |
+  +-----------------------------------+-----------------+
+  | `Kirstie Whitaker`_ (@KirstieJane)| 0.5h/week       |
+  +-----------------------------------+-----------------+
+
+
+Rules
+`````
+
+1. Potential modifications to the Repository should first be proposed via an 
+   Issue. Rules regarding Votes apply to both Pull Requests and Issues.
+
+   - Every modification of the specification (including a correction of a typo, adding a new Contributor, an extension adding support for a new data type, or others) or proposal to release a new version needs to be done via a Pull Request (PR) to the Repository.
+2. Anyone can open a PR (this action is not limited to Contributors).
+3. PRs adding new Contributors must also add their GitHub names to the 
+   `all-contributors file`_. 
+   This should be done with the allcontributors bot.
+
+   - Contributors may also add themselves to the Zenodo file if they wish, but this is not mandatory.
+4. A PR is eligible to be merged if and only if these conditions are met:
+
+   a) The PR features at least two `Reviews that Approve <https://help.github.com/articles/about-pull-request-reviews/#about-pull-request-reviews>`_
+      the PR from Maintainers of which neither is the author of the PR. 
```

Comment:
Do the approvals _have_ to be from Maintainers ? Personally, I'm fine if approvals come from any community member.

Reply:
I'm not wild about any community member approving stuff, I would feel more warm and cozy knowing that maintainers are approving. Honestly, though, I just doubt that difference will come up much. Anyone who's approving reviews often and doing a decent job will de facto become a maintainer. So perhaps we should go with the more open model of community members being allowed to approve.

Reply:
I'd rather give anyone the option to review & approve. We've already had one case on BIDS derivatives where we asked for a BIDs developer who isn't non-tedana person to review. (He didn't, but it's still a use case for this scenario).
With the proposed system, the steering committee can raise the bar for any PR so I'd rather keep it so that the default is any contributor can review, but the steering committee can change that.

Reply:
Also, given how long it takes for our current Maintainers to review some of the harder PRs, if an enthusiastic contributor comes along and wants to do a serious review, I'm all for it!